src/XRP/issued-currency-client.ts

```diff
@@ -81,4 +81,34 @@ export default class IssuedCurrencyClient {
       wallet,
     )
   }
+
+  /**
+   * Enable Default Ripple for this XRPL account.
+   *
+   * @see https://xrpl.org/become-an-xrp-ledger-gateway.html#default-ripple
+   *
+   * @param wallet The wallet associated with the XRPL account enabling Default Ripple and that will sign the request.
+   * @returns A promise which resolves to a TransactionResult object that contains the hash of the submitted AccountSet transaction,
```

Comment:
i think it's alright to let the TransactionResult abstraction do the heavy lifting here...maybe something like `...a TransactionResult object that represents the result of this transaction` (or something less tautological lmao)